src/base/rss/rssfeed.h

```diff
@@ -65,6 +65,7 @@ namespace Rss
 
     public:
         Feed(const QString &url, Manager *manager);
+        explicit Feed(const QString &url);
```

Comment:
IMO, it's better to leave only one constructor here (with default second param):
```c++
explicit Feed(const QString &url, Manager *manager = nullptr);
```